webapp/views/App/views/Data/MapView/CoordinateAttributeDataLayer/CoordinateAttributePolygon.js

```diff
@@ -9,24 +9,24 @@ import PropTypes from 'prop-types'
 
 export const CoordinateAttributePolygon = (props) => {
   const surveyInfo = useSurveyInfo()
-  const polygonProperties = JSON.parse(surveyInfo.props.samplingPolygon)
+  const polygonProperties = surveyInfo.props.samplingPolygon || {}
```

Comment:
there is a getter in Survey for this, right? Survey.getSamplingPolygon, am I wrong?